testsuite/features/secondary/trad_migrate_to_minion.feature

```diff
@@ -81,6 +81,10 @@ Feature: Migrate a traditional client into a Salt minion
     When I wait until file "/tmp/remote-command-on-migrated-test" exists on "sle_migrated_minion"
     And I remove "/tmp/remote-command-on-migrated-test" from "sle_migrated_minion"
 
+  Scenario: Cleanup: remove package from the migrated SSH minion
```

Comment:
This is the same scenario name found at testsuite/features/secondary/trad_migrate_to_sshminion.feature
Shouldn't be scenario names unique to facilitate debugging?

Reply:
whoops, you are completly right, I'll fix it

thanks for the catch

Reply:
Fixed